Data/models/Bookmark.swift

```diff
@@ -425,6 +426,55 @@ extension Bookmark {
         record?.delete()
     }
     
+    /// Gets all nested bookmarks recursively.
+    public static func getAllChildren(forFolderUUID syncUUID: [Int]?,
```

Comment:
This looks pretty similar to `getChildren`, can we centralize logic any way?

Reply:
Not easy to centralize it without nesting logic, for `getAllChildren` we never want to ignore folders while `getChildren` has this option. And recursive search never wants to skip folders.

I think of renaming it to `getAllInsideOf(folderUUID syncUUID..`

Reply:
Oh this is recursive? If so, should be renamed, I think `getAllInsideOf` is probably better, helps. Or `getRecursiveChildren` or something.